sources/packages/web/src/components/layouts/BodyHeaderContainer.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div>
+    <!-- Header slot (with and without v-card) -->
+    <v-card v-if="card"><slot name="header"></slot></v-card>
```

Comment:
I didn't get this part. The v-card or div must be at the parent/root level for both header and body right?

Reply:
Yes, I have corrected it now.